app/helpers/job_application_helper.rb

```diff
@@ -98,4 +98,13 @@ def job_application_build_submit_button_text
       t("buttons.save_and_continue")
     end
   end
+
+  def job_application_view_applicant(vacancy, job_application)
+    if job_application.withdrawn?
+      tag.span "#{job_application.first_name} #{job_application.last_name}", class: "govuk-!-font-size-19"
```

Comment:
There's a `job_application.name` waiting to happen 😁

Reply:
I've been thinking the same thing for a little while now lol. I'll make a PR